### PR TITLE
New template for simple devtest environment

### PR DESCRIPTION
**Pull request by adhurwit**
### Contributing guide
https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md

### Changelog
*
*
*
*
*

### Description of the change

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@adhurwit__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (adhurwit). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by adhurwit** [comment]
#ready-to-merge #sign-off

File: devtest-p2s-iis/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+         "metadata":{  
+            "description":"Client root certificate data used to authenticate VPN clients."
+         }
+      }
+      
+   },
+   "variables":{  
+      "location":"[resourceGroup().location]",
+      "apiVersion":"2015-06-15",
+      "newStorageAccount": "[concat('store',uniqueString(resourceGroup().id))]",
```
Comment (singhkay) [inline devtest-p2s-iis/azuredeploy.json]: Do the unique string first, helps spread storage accounts. See #944

File: devtest-p2s-iis/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+      "appSubnetPrefix": "10.0.0.0/24",
+      "appSubnetRef": "[concat(variables('vnetID'),'/subnets/',variables('appSubnetName'))]",
+      "nicName": "[concat(parameters('environmentPrefix'),'NIC01')]",
+      "vmName": "[concat(parameters('environmentPrefix'),'VM01')]",
+      "vmSize": "Standard_D1",
+      "imagePublisher": "MicrosoftWindowsServer",
+      "imageOffer": "WindowsServer",
+      "imageSKU": "2012-R2-Datacenter",
+      "vmExtensionName": "dscExtension",
+      "modulesUrl":"https://github.com/adhurwit/azure-quickstart-templates/raw/master/devtest-p2s-iis/DevTestWebsite.ps1.zip",
```
Comment (singhkay) [inline devtest-p2s-iis/azuredeploy.json]: Should point to this repo

**Comment by adhurwit** [comment]
it failed because I updated the path for the download file...

**Comment by singhkay** [comment]
@adhurwit It failed because the links need to be in this format https://github.com/Azure/azure-quickstart-templates/tree/master/1-CONTRIBUTION-GUIDE#rawgithubusercontentcom-links

When they are in that format, the CI will replace them automatically to your fork

**Comment by adhurwit** [comment]
@singhkay Ah, OK - thanks. Didn't see change in guide. Update. #sign-off

**Comment by singhkay** [comment]
lgtm, merged

**Pull request merged**